share/module_check_a_mundo.F

```diff
@@ -1591,6 +1591,30 @@ END FUNCTION bep_bem_ngr_u
          CALL wrf_debug ( 1, wrf_err_message )
       END IF
 
+!-----------------------------------------------------------------------
+! Make sure phot_blcld is only used when icloud_bl==1 and  MYNN is chosen.
+!-----------------------------------------------------------------------
+
+      oops = 0
+      DO i = 1, model_config_rec % max_dom
+         IF ( .NOT. model_config_rec % grid_allowed(i) ) CYCLE
+         IF ( model_config_rec%phot_blcld(i) .EQV. .TRUE. ) THEN
```

Comment:
@jordanschnell 
I don't see "phot_blcld" defined in the registry. Is this supposed to be "phot_opt==tuv"?

Reply:
@davegill It is supposed to be the new namelist option phot_blcld added in registry.chem - is this not the correct way to do this?